src/core/Core.tsx

```diff
@@ -42,6 +44,11 @@ export const Core = () => {
       await dispatch(unlockApp({ isOffline, initializeWallet })).unwrap()
     } catch (err) {
       console.log('ERR CORE', err)
+      if (err instanceof TraceError) {
```

Comment:
Why did you setError which you don't use?

Reply:
Maybe better to set global error?

Reply:
I'm confused here, since this is a developer error, do we need to handle it here for the user? Can the error be thrown where it was and not caught that way the developer needs to address it?

Reply:
I changed. I use set global error to show the error.